src/Enterspeed.Source.UmbracoCms.Shared.NetCore/Enterspeed.Source.UmbracoCms.Shared/EnterspeedPropertyValidation/PropertyValidationResponse.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Enterspeed.Source.UmbracoCms.Shared.EnterspeedPropertyValidation
+{
+    public class PropertyValidationResponse : IEnterspeedValidationResponse
+    {
+        public PropertyValidationResponse(string validationError, bool valid)
+        {
+            ValidationMessage = validationError;
+            Valid = valid;
+        }
+
+        public bool Valid { get; }
+        public string ValidationMessage { get; }
+    }
+
+    public interface IEnterspeedValidationResponse
```

Comment:
Should we split the interface and the class into two files?

Reply:
Yes. To be honest I have been setting this up quite fast to start a dialog around the approach :)

Reply:
That's totally fine, np :-)